coordinator/src/test/scala/filodb.coordinator/client/SerializationSpec.scala

```diff
@@ -41,7 +41,13 @@ class SerializationSpec extends ActorTest(SerializationSpecConfig.getNewSystem)
     serializer.fromBinary(serializer.toBinary(thing))
   }
 
-  it("should be able to serialize IngestionConfig, SetupDataset, DatasetResourceSpec, IngestionSource") {
+  val dummyFailureProvider = new FailureProvider {
```

Comment:
Call this emptyFailureProvider and reuse in all places where created again.

Reply:
Changed to emptyFailureProvider